srcpkgs/openimagedenoise/template

```diff
@@ -12,7 +12,11 @@ maintainer="teldra <teldra@rotce.de>"
 license="Apache-2.0"
 homepage="https://openimagedenoise.github.io"
 distfiles="https://github.com/OpenImageDenoise/oidn/releases/download/v${version}/oidn-${version}.src.tar.gz"
-checksum=948b070c780b5de0d983e7d5d37f6d9454932cc278913d9ee5b0bd047d23864a
+checksum=88367b2bbea82d1df45d65141c36b6d86491bc6b397dc70beb3a05dda566f31c
+
+do_check() {
+	:
+}
```

Comment:
Please use `make_check=no` or `make_check=extended`, and leave a comment explaining why.

Reply:
Done